projects/github-actions/repo-gardening/src/tasks/triage-issues/ai-labeling.js

```diff
@@ -94,6 +94,27 @@ Example response format:
 	return { labels, explanations: parsedResponse };
 }
 
+/**
+ * Clean up the issue content for OpenAI processing.
+ * Remove links from the issue content.
+ *
+ * @param {string} content - Issue body content.
+ *
+ * @return {string} Cleaned up issue content.
+ */
+function cleanIssueContent( content ) {
+	// Remove links in the format [link text](url), but keep the link text.
+	content = content.replace( /\[(.*?)\](?:\([^)]+\))?/g, '$1' );
```

Comment:
If someone does a link like

> [Link text][1] and [another link][example.org]
>
> [1]: https://example.com
> [example.org]: https://example.org

```md
[Link text][1] and [another link][example.org]

[1]: https://example.com
[example.org]: https://example.org
```
it'll turn into
```
Link text1 and another linkexample.org

1: this link with more information
example.org: this link with more information
```

Also worth noting is that without the corresponding "footnote" it doesn't get linkified at all:

> [just brackets]

> [broken link][2]

Then there's the possibility of stuff like this

<details><summary>

> [huh][what](https://example.com)

</summary>

```md
[huh][what](https://example.com)
```

</details>

and

<details><summary>

> [huh][what][ok] [huh][ok][what]
>
> [ok]: https://example.com

</summary>

```md
[huh][what][ok] [huh][ok][what]

[ok]: https://example.com
```

</details>

Reply:
I'm thinking we may be okay leaving those edge cases out, since that's not format that will commonly used in our issues here at Automattic, or overall in GitHub issues imo.